### ISSUE TITLE
Not a limit raise 

### ISSUE THREAD

**Issue by ThorvaldAagaard** [EdwardPiwowar/BBA#634]
000C9F694BC836E4CBC54F2F34C6
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/c6916b1f-29bf-49bd-b8df-a2dd35151be4)
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/a7347295-e759-4b92-b669-0f16948192d4)

**Comment by ThePokerDude** [comment]
So what does 2s mean? I did not find any description.
But "impossible" bids usually show maximum with fit.

**Comment by ThorvaldAagaard** [comment]
Yes, and it has been added if opener bids a minor, but over 2H, south can bid 3H (or 4H).
Inventing agreements is bad for robots

**Comment by EdwardPiwowar** [comment]
Next BBA has an "Imposible 2S" option.
